tests/integration_tests/security/test_jail.py

```diff
@@ -234,7 +233,7 @@ def sys_setup_cgroups():
     This set-up is important to do when running from inside a Docker
     container while the system is using cgroup-v2.
     """
-    cgroup_version = 1 if cgroup_v1_available() else 2
+    cgroup_version = 2 if cgroup_v2_available() else 1
```

Comment:
Doesn't this flip the default to cgroupsv2 if both are available? Though IMO we should definitely prefer v2 if both are available.

Reply:
Good point. However, I'm not quite sure that the behavior is flipped regarding what version we prefer because I don't think we were in the situation to have both versions enabled in parallel (correct me if I'm wrong since I don't have to much experience with this). Up until now, we had to provide the `cgroup_no_v1=all` flag to the cmdline in order to enable cgroupv2 (which keeps v1 disabled) so I don't think we were in the situation to have both versions enabled in parallel. This means that when we had v2 enabled we simply used it which is what should happen after this PR.